settings.gradle

```diff
@@ -0,0 +1,4 @@
+plugins {
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.8.0'
```

Comment:
Author note:
this allows the gradle toolchain mechanism to automatically download and use  the required jdk version if missing locally